lib/screens/todo/todo_page.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+
+import '../../models/task.dart';
+
+class TODOPage extends StatefulWidget {
```

Comment:
```suggestion
class TaskPage extends StatefulWidget {
```
Вы же в остальных местах задачу называете `Task`